bindings/nodejs/generated.js

```diff
@@ -256,6 +256,35 @@ switch (platform) {
           loadError = e
         }
         break
+      case 'riscv64':
```

Comment:
How can this change related to package manager change?

Reply:
The version of napi-cli has changed because of lock file turnover. This file is generated as a result.